core/src/tasks/recordProperty/importRecordProperty.ts

```diff
@@ -70,7 +73,36 @@ export class ImportRecordProperty extends RetryableTask {
       hash[property.id] = Array.isArray(propertyValues)
         ? propertyValues
         : [propertyValues];
-      await record.addOrUpdateProperties(hash);
+
+      try {
+        await record.addOrUpdateProperties(hash);
+      } catch (error) {
+        if (property.unique && error instanceof UniqueConstraintError) {
```

Comment:
anything else we should catch?

Reply:
I don't want to suppress /all/ the errors yet, and I still want to see a resque error for something unexpected so we can choose what do to about it.  

That said, I think there are 2 classes of thing we catch now:
* validation errors from our code (eg: that's not an email address)
* uniqueness errors from the database (this PR)

Other than networking errors, there are other classes of database error which might popup (bad encoding? string too long?) that I really do want to throw